blueocean-config/src/main/resources/io/jenkins/blueocean/config/BlueOceanConfig/header.jelly

```diff
@@ -0,0 +1,18 @@
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
+  <script>
+    (function () {
+      window.$$blueoceanConfig = JSON.parse('${it.blueOceanConfig}');
```

Comment:
> @tfennelly I don't have context on jenkinsCIGlobal. So how does other plugin config is added to it? Please post pointer to doc where other plugins add data to it.

The only example is in `js-modules`. None of the other plugins put anything into the global scope.

> Sorry, not sure what you mean by this. You mean blueocean-config injects a element with some id and it gets picked up by js-extensions (?)?

Right.